src/components/Layout/Header.js

```diff
@@ -44,37 +46,45 @@ const Header = () => {
-          <Grid container item xs={9}>
-            <Button component={Link} to='/admin-app/students' color='inherit' className={classes.btn}>
-              Студенты
-            </Button>
-            <Button component={Link} to='/admin-app/groups' color='inherit' className={classes.btn}>
-              Группы
-            </Button>
-            <Button component={Link} to='/admin-app/payments' color='inherit' className={classes.btn}>
-              Платежи
-            </Button>
-            <Button component={Link} to='/admin-app/teachers' color='inherit' className={classes.btn}>
-              Учителя
-            </Button>
-            <Button aria-controls='simple-menu' aria-haspopup='true' onClick={handleClick} color='inherit'>
-              Расписание
-            </Button>
-            <Menu id='simple-menu' anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleClose}>
-              <MenuItem component={Link} to='/admin-app/lessons' onClick={handleClose}>
-                Раписание по Группам
-              </MenuItem>
-              <MenuItem component={Link} to='/admin-app/teachers/lessons' onClick={handleClose}>
-                Расписание по Учителям
-              </MenuItem>
-            </Menu>
-          </Grid>
-          <Grid item>
+          </Box>
+          <Box flexGrow={1} display={user ? 'block' : 'none'}>
+            {isAdminRole ? (
+              <>
+                <Button component={Link} to='/admin-app/students' color='inherit' className={classes.btn}>
+                  Студенты
+                </Button>
+                <Button component={Link} to='/admin-app/groups' color='inherit' className={classes.btn}>
+                  Группы
+                </Button>
+                <Button component={Link} to='/admin-app/payments' color='inherit' className={classes.btn}>
+                  Платежи
+                </Button>
+                <Button component={Link} to='/admin-app/teachers' color='inherit' className={classes.btn}>
+                  Учителя
+                </Button>
+                <Button aria-controls='simple-menu' aria-haspopup='true' onClick={handleClick} color='inherit'>
+                  Расписание
+                </Button>
+                <Menu id='simple-menu' anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={handleClose}>
+                  <MenuItem component={Link} to='/admin-app/lessons' onClick={handleClose}>
+                    Раписание по Группам
+                  </MenuItem>
+                  <MenuItem component={Link} to='/admin-app/teachers/lessons' onClick={handleClose}>
+                    Расписание по Учителям
+                  </MenuItem>
+                </Menu>
+              </>
+            ) : (
+              <Button component={Link} to='/admin-app/teachers/lessons' color='inherit'>
+                Расписание
+              </Button>
+            )}
+          </Box>
+          <Box>
             {user ? (
```

Comment:
Копка не нужна  если не залогинен user, итак только экран авторизации выводится
```suggestion
         {user && (
```

Reply:
Ты про какую кнопку? Все кнопки не видны если пользователь не залогинен.

Reply:
убрал кнопку войти